scripts/apps/search/components/ItemList.jsx

```diff
@@ -29,6 +31,10 @@ export class ItemList extends React.Component {
         this.setSelectedComponent = this.setSelectedComponent.bind(this);
         this.modifiedUserName = this.modifiedUserName.bind(this);
         this.setNarrowView = this.setNarrowView.bind(this);
+
+        keyboardManager.bind('x', () => {
```

Comment:
Thanks Gabriel. That did work. I really should have tried that out before resubmitting this PR though - my bad. Cheers.